pint_xarray/testing.py

```diff
@@ -0,0 +1,22 @@
+from . import conversion
+
+
+def assert_units_equal(a, b):
+    """ assert that the units of two objects are equal
```

Comment:
```suggestion
    """ assert that the units of two xarray objects are equal
```
Might be good to clarify in the summary line that this is just for xarray objects?